### DISCUSSION TITLE
Does KPN TV+ still needs multicast routing and IGMP-Proxy?

### DISCUSSION THREAD

**Discussion by michaelblaas** [fabianishere/udm-iptv#227] [Q&A]
Hi all,

The Dutch provider KPN switched from 'Interactieve TV' to KPN TV+ (see: [https://www.kpn.com/televisie.htm]). As far as I understand the new 'settop box' is kind of a Chromecast/AndroidTV device.

Does anyone have them installed already and, more important: do these still need the multicast routing protocol and the IGM-Proxy to work? After all: a normal AndroidTV with the KPN TV-app works on a simple internet connection...

Hope someone has tested this...
Regards
Michael

**Comment by valuta1995** [comment]
Hi Michael,

I just moved last week and my old Arris KPN STB is no longer functioning on VLAN 4 (IPTV), but instead only works if I connect it on VLAN 6 (internet). I'm not well versed enough to more precisely describe the issue without some help. But I think it may be related.

Update: Just rang up KPN support, apparently VLAN 6 is now good for the STB, no more special configuration required.

Sincerely,
Glenn

**Comment by avwolferen** [comment]
@michaelblaas @valuta1995 

This made live easier for my family :)
After the pre-previoud UDM Pro update a coupe of months ago I was forced to pause/play after every channel switch, first 3 seconds the multicast stream was totally fine but then froze. Couldn't make it work properly again and accepted the pause/play workaround. Two days ago my UDM Pro updated and I had to reinstall the package. Saw that there was also LAN available where to make IPTV available. I am 100% sure I had the VLAN configuration properly working. Nevertheless it works like a charm again!

@fabianishere keep up the amazing work!

Best regards,
Alex

**Comment by rcjkierkels** [comment]
We just received KPN+ TV (Before we had Ziggo) and I only needed to setup the internet connection part. I do not need a VLAN 4 to watch TV. It just works out of the box. I had to restart my UDM Pro after updating the WLAN settings but thats all. After that I do have internet and TV working. No need to install anything anymore.

**Comment by hhuitema** [comment]
The New KPN TV+ box works both with Unicast and Multicast traffic. The main difference is that the UniCast stream takes longer to build up. (up to 4 seconds) and the Stream itself has a lower bitrate.
When you use the Multicast streams (through VLAN 4) quality is higher and the the zaptimes are a bit quicker.

So yes KPN IP TV works without VLAN4, but for the best experience....... :-)
